1-mandatory/1-writers.js

```diff
@@ -62,7 +65,11 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+   writers.map(({firstName, lastName, age, alive}) => {
+    if ((age >= 40) && (age <= 49) && !alive) {
+      console.log(`Writer ${firstName} ${lastName} died at ${age} years old.`)
+    }
+  });
 }
```

Comment:
Comment: Well done

Consider: Here is another way to do it.
function logAllWriters() {
// write your code to log all writers here
for (let i = 0; i < writers.length; i++) {
const writer = writers[i];
console.log(Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.);
}
}